### PR TITLE
Front advert detail page

### PR DESCRIPTION
**Pull request by pKraszewska**
here is new advert detail page

### CODE REVIEW

File: front/src/App.js [hunk 1]
```diff
@@ -3,11 +3,12 @@ import Header from "./header/Header";
 import HeaderSearch from "./header/HeaderSearch";
 import News from "./mainContent/News";
 import Footer from "./header/Footer";
-import LoggedSidebar from "./sideBard/LoggedSidebar";
+import LoggedSidebar from "./sideBar/LoggedSidebar";
```
Comment (Ocieplino) [inline front/src/App.js:6]: Im happy that we dont have mr. Bard anymore 👌

File: front/src/sideBar/AdvertPage.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+      advert_posted_on: data.posted_on,
+      advert_genre: data.genre,
+      advert_profession: data.profession
+    });
+  }
+
+  render() {
+    return (
+      <Container className="news-container">
+        <Flexbox
```
Comment (Ocieplino) [inline front/src/sideBar/AdvertPage.js]: I think it should be in external style sheet

File: front/src/sideBar/AdvertPage.js [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+              {this.state.advert_profession}
+            </Tag>
+            <a className="advert-header" /> GENRE:
+            <Tag color="orange" className="advert-tag">
+              {this.state.advert_genre}
+            </Tag>
+          </TagGroup>
+        </Flexbox>
+        <Divider />
+        <Flexbox
```
Comment (Ocieplino) [inline front/src/sideBar/AdvertPage.js]: Yep, here too, then you dont have repeted code.

File: front/src/sideBar/AdvertPage.js (hunk 2 above)
Comment (Ocieplino) [reply front/src/sideBar/AdvertPage.js]: I mean flexbox

**Review by sznowak13 — APPROVED** [review]
I approve. :+1:

**Pull request merged**